src/less/success-message/base/success-message.less

```diff
@@ -0,0 +1,53 @@
+.success-message--full {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+    position: relative;
+
+    .fake-btn,
+    .btn {
+        bottom: 0;
+        left: 0;
+        position: absolute;
+        width: 100%;
+    }
+}
+
+.success-message {
+    background-color: @success-message-background-color;
+    box-sizing: border-box;
+    padding: 0 9%;
+    text-align: center;
+    width: 100%;
+
+    .fake-btn,
+    .btn {
+        background-color: @success-message-button-color;
+        border-color: @success-message-button-border-color;
+        margin-top: 28px;
+
+        &:hover {
+            background-color: @success-message-button-color-hover;
+            border-color: @success-message-button-color-hover;
+            color: @success-message-button-border-color;
+            text-decoration: none;
+        }
+    }
+}
+
+.success-message__title {
+    .background-image-base64(@icon-confirmation-filled-base64);
+    background-position: center top;
+    background-repeat: no-repeat;
+    background-size: 35px 35px;
+    font-size: @font-size-30;
+    font-weight: bold;
+    margin: 0;
+    padding-top: 54px;
```

Comment:
I was wondering why the padding was so high on this. And I see it was because the background image was part of the h2. 
I think we should split it up somehow. We should have a container/element for the background image, and then another container/element below it with the text. This also makes it easier if the user wants to use foreground images (since we might want to switch to those by default sometime in the future)

Reply:
I agree with @agliga. We get more control in terms of accessibility that way too (i.e. we can add aria attributes if necessary).

Reply:
@agliga  I thought that originally, but went with this because there isn't any variation allowed on the icon (other than maybe changing it to a foreground icon eventually).  So this reduces the complexity.
@ianmcburnie  I was wondering if this was accessible.  But I thought that it would be OK since the title section is required and would be the target for the "aria-labelledby".
But if you think we need the extra element, I can make that change when I'm back on Thursday.

Reply:
Technically this is accessible as it stands, because there is plenty of nearby text that conveys that this was a "success". I just think that having a standalone element such as a span buys us a bit more flexibility. Besides, we always need a standalone element for foreground svg.